boards/arm/google_kukui/Kconfig.board

```diff
@@ -0,0 +1,10 @@
+# Kconfig - Google Kukui EC
+#
+# Copyright 2019 The Chromium OS Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
Could shorten like this (see https://github.com/zephyrproject-rtos/zephyr/pull/20280 and https://docs.zephyrproject.org/latest/guides/kconfig/index.html#header-comments-and-other-nits):

```
# Google Kukui EC

# Copyright 2019 The Chromium OS Authors
# SPDX-License-Identifier: Apache-2.0
```